TurkishFilter/sections/css_extended.txt

```diff
@@ -17,6 +17,7 @@ telesikayet.com#?#.CloudWrapper > div[align="center"] > span.previous_results_cl
 vipingilizce.net#?#.ContentAnasayfa > fieldset > legend:contains(/^Reklam$/)
 cimri.com#?#table > tbody > tr:has(div[width="728"][height="90"])
 letgo.com#?#div[class^="Container"] > div[class^="Spacer"] div[id^="div-gpt-ad"]:upward(1)
+jojdn2.xyz#?##oframeplayerjs > pjsdiv[style]:has(> pjsdiv[style*="width: 318px;"])
```

Comment:
`width` is different on my end. Better try `background: rgb(255, 0, 0);` (check in app and ext)